Formula/buildozer.rb

```diff
@@ -14,11 +14,96 @@ class Buildozer < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "a257921eb9df552c485cb2656ce7f535a708846308de4aa1ee47942492a61aee"
   end
 
-  depends_on "bazelisk" => :build
+  depends_on "go" => :build
 
   def install
-    system "bazelisk", "build", "--config=release", "buildozer:buildozer"
-    bin.install "bazel-bin/buildozer/darwin_amd64_stripped/buildozer"
+    ENV["GOPATH"] = buildpath
```

Comment:
This shouldn't be needed in modern projects

Reply:
fixed in https://github.com/Homebrew/homebrew-core/pull/70924/commits/2ad2cd420b0270a68d90a320dfcf91fa36374ee3